### PR TITLE
Comparison results table

### PR DESCRIPTION
**Pull request by ianatesfa**

### CODE REVIEW

File: src/SFA.DAS.FindEmploymentSchemes.Web/Views/Schemes/ComparisonResults.cshtml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+﻿@model IEnumerable<Scheme>
+
+@section BreadCrumbs {
+    <a href="/" class="govuk-back-link">Home</a>
```
Comment (lazcool) [inline src/SFA.DAS.FindEmploymentSchemes.Web/Views/Schemes/ComparisonResults.cshtml:4]: This should go back to the 'pick schemes' page.

File: src/SFA.DAS.FindEmploymentSchemes.Web/Views/Schemes/ComparisonResults.cshtml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+                            @scheme.ShortBenefits
+                        </td>
+                        <td class="govuk-table__cell">
+                            @scheme.ShortTime
+                        </td>
+                    </tr>
+                }
+            </tbody>
+        </table>
+    </div>
```
Comment (lazcool) [inline src/SFA.DAS.FindEmploymentSchemes.Web/Views/Schemes/ComparisonResults.cshtml:48]: We need a 'Return to list of all schemes' link under the table.

File: src/SFA.DAS.FindEmploymentSchemes.Web/Views/Schemes/ComparisonResults.cshtml (hunk 1 above)
Comment (ianatesfa) [reply src/SFA.DAS.FindEmploymentSchemes.Web/Views/Schemes/ComparisonResults.cshtml]: Done in latest commit

File: src/SFA.DAS.FindEmploymentSchemes.Web/Views/Schemes/ComparisonResults.cshtml (hunk 2 above)
Comment (ianatesfa) [reply src/SFA.DAS.FindEmploymentSchemes.Web/Views/Schemes/ComparisonResults.cshtml:51]: Done in latest commit

File: src/SFA.DAS.FindEmploymentSchemes.Web/Views/Schemes/ComparisonResults.cshtml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ﻿@model IEnumerable<Scheme>
 
 @section BreadCrumbs {
-    <a href="/" class="govuk-back-link">Home</a>
+    <a href="javascript:history.back()" class="govuk-back-link">Back</a>
```
Comment (lazcool) [inline src/SFA.DAS.FindEmploymentSchemes.Web/Views/Schemes/ComparisonResults.cshtml:4]: Can we use asp-route, as this needs to work without js.

Comment (ianatesfa) [reply src/SFA.DAS.FindEmploymentSchemes.Web/Views/Schemes/ComparisonResults.cshtml]: OK, changed back to /scheme-comparison

**Review by lazcool — APPROVED** [review]

**Pull request merged**